Embedded Metadata.js

```diff
@@ -688,7 +724,7 @@ function addLowQualityMetadata(doc, newItem) {
 		}
 	}
 	// fall back to "keywords"
-	if (!newItem.tags.length) {
+	if (newItem.tags.length == 0) { // this needs to be tested; previously was !...length, which evaluated to true when zero
```

Comment:
this might need to be tested; previously was `!newItem.tags.length`, which evaluated to true when zero

appeared to work fine on https://www.bloomberg.com/news/articles/2019-09-12/peloton-founder-goes-from-kickstarter-to-a-450-million-fortune